### PR TITLE
Fixing compilation errors due to changes to the `logging` interface.

### PR DESCRIPTION
**Pull request by asridharan**
Fixed tests.

---
This change is [Reviewable](https://reviewable.io/reviews/dcos/dcos-mesos-modules/31)

### CODE REVIEW

**Comment by asridharan** [comment]
@GoelDeepak @kaysoky can you do a quick review on this. After these changes the tests compile and pass. 

Apart from the compilation error also needed to fix some of the test to take into account changes to the CNI configuration that was introduced to support port-mapping with DC/OS overlay.

File: tests/overlay_tests.cpp [hunk 1]
```diff
@@ -570,22 +570,22 @@ TEST_F(OverlayTest, ROOT_checkMesosNetwork)
   ASSERT_SOME(json);
 
   Result<JSON::String> network = json->find<JSON::String>("name");
-  ASSERT_SOME(network);
+  ASSERT_SOME(network) ;
```
Comment (kaysoky) [inline tests/overlay_tests.cpp]: Extraneous space :smiley:

File: tests/overlay_tests.cpp [hunk 2]
```diff
@@ -570,22 +570,22 @@ TEST_F(OverlayTest, ROOT_checkMesosNetwork)
...
 
-  Result<JSON::Number> mtu = json->find<JSON::Number>("mtu");
-  ASSERT_SOME(mtu);
+  Result<JSON::Number> mtu = json->find<JSON::Number>("delegate.mtu");
+  ASSERT_SOME(mtu) << "CNI config: " << cniConfig.get();
   EXPECT_EQ(mtu.get(), 1420);
 
-  Result<JSON::Object> ipam = json->find<JSON::Object>("ipam");
-  ASSERT_SOME(ipam);
+  Result<JSON::Object> ipam = json->find<JSON::Object>("delegate.ipam");
```
Comment (kaysoky) [inline tests/overlay_tests.cpp]: Can you add a link to where this config was changed?

Presumably these fields are now found in a deeper JSON object layer now
`"delegate" : { "ipMasq" : ... }`

**Comment by asridharan** [comment]
@kaysoky addressed your comments.

**Pull request merged**